src/logger.js

```diff
@@ -24,7 +32,7 @@ function log (channel, lvl, ...args) {
   handler(timestamp, channel, lvl, ...args)
   if (OPTIONS.alsoLogInLocalConsole) {
     const variant = LoggerVariants[lvl]
-    variant(channel, ...args)
+    variant.log(variant.prefix, channel, ...args)
```

Comment:
This makes it clearer what type of message is shown in the console

Reply:
Yeah that makes sense, because electron's chrome doesn't show the icons that as example Firefox does